test/unit/helpers/bpmn-model-utils.ts

```diff
@@ -88,36 +88,32 @@ const newGroup = (parent: string, id: string, name: string): Shape => new Shape(
 const newSubProcess = (parent: string, id: string, name: string, isExpanded: boolean): Shape =>
   new Shape(buildShapeId(id), new ShapeBpmnSubProcess(id, name, ShapeBpmnSubProcessKind.EMBEDDED, parent, isExpanded ? undefined : [ShapeBpmnMarkerKind.EXPAND]));
 
-const addContainerElements = (bpmnModel: BpmnModel, containerWithLanes: ContainerWithLanes & BaseElement): void => {
+const addContainerElements = (bpmnModel: BpmnModel, containerWithLanes: ContainerWithLanes & BaseElement, parentId: string): void => {
```

Comment:
Let's discuss about the method signature change.
See  https://github.com/process-analytics/bpmn-visualization-js/pull/2129#discussion_r930014938